multiuser_service_windows.go

```diff
@@ -0,0 +1,295 @@
+// +build multiuser
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+	"time"
+
+	"golang.org/x/sys/windows"
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/debug"
+	"golang.org/x/sys/windows/svc/eventlog"
+	"golang.org/x/sys/windows/svc/mgr"
+
+	"gopkg.in/natefinch/lumberjack.v2"
+)
+
+// stderr handle is invalid when run as service
+var logWriter = io.MultiWriter(ioutil.Discard)
```

Comment:
What is the reason to create an `io.MultiWriter` that wraps two writers, where one of them is always `ioutil.Discard`?

Reply:
```suggestion
var logWriter = io.MultiWriter()
```